test/e2e/helm_chart_test.go

```diff
@@ -298,14 +297,11 @@ func deleteDeploymentAndOperator(data *model.TestDataProvider) {
 	By("Check project, deployment does not exist", func() {
 		helm.Uninstall(data.Resources.Deployments[0].Spec.GetDeploymentName(), data.Resources.Namespace)
 		Eventually(
-			func(g Gomega) {
-				if atlasClient.IsProjectExists(g, data.Resources.ProjectID) {
-					_, err := atlasClient.Client.Projects.Delete(context.TODO(), data.Resources.ProjectID)
-					g.Expect(err).To(BeNil())
-				}
+			func(g Gomega) bool {
+				return atlasClient.IsProjectExists(g, data.Resources.ProjectID)
 			},
 			"7m", "20s",
-		).Should(Succeed(), "Project and deployment should be deleted from Atlas")
+		).Should(BeFalse(), "Project and deployment should be deleted from Atlas")
```

Comment:
An revert this change:
```suggestion
		).Should(Succeed(), "Project and deployment should be deleted from Atlas")
```